spec/features/two_factor_authentication/sign_in_spec.rb

```diff
@@ -156,12 +144,13 @@
 
       scenario 'allows a user to continue typing even if a number is invalid', :js do
         sign_in_before_2fa
-        select 'United States of America +1', from: 'International code'
+        find('.selected-flag').click
```

Comment:
Can we encapsulate this logic in a method that takes the country to select and performs the finding and clicking? This can be located in the same file for now, but at some point we may want it shared if we have to access it for other phone number change related tests.

Reply:
On it, thanks!